scripts/benchmarks/benchmark.py

```diff
@@ -503,15 +515,15 @@ def write_result(self):
         if os.path.isfile(self.gpu_log_filename):
             # Add GPU info and measurements into the result saving
             peak_mem_usage_by_device_id, device_name = self.get_peak_mem_usage_by_device_id()
-            save_result[REPORT_DEVICE_FIELD_NAME] = device_name
+            save_result[RESULT_FIELD_DEVICE_NAME] = device_name
             # Memory usage is averaged across all devices in the final result
-            save_result[REPORT_GPU_FIELD_NAME] = peak_mem_usage_by_device_id.mean()
+            save_result[RESULT_FIELD_RESERVED_GPU_MEM] = peak_mem_usage_by_device_id.mean()
 
         # process gpu mem from output metrics and write to result
         if save_result.get(HF_ARG_SKIP_MEMORY_METRIC) == "False":
```

Comment:
this will give problem if `HF_ARG_SKIP_MEMORY_METRIC` is not specified in the arguments.